### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by rdthetha**
Alert component

### CODE REVIEW

File: component.css [hunk 1]
```diff
@@ -3,13 +3,18 @@
...
+@import "./components/alert/alert.css";
 @import "./index.css";
 
 :root{
 
     --primary-background:#E2F1EC;
     --primary-color:#005C68;
     --primary-font:'Spartan', sans-serif;
+    --primary-faint-red:rgb(226, 124, 124);;
+    --primary-faint-green:#99e696;
```
Comment (anasm11) [inline component.css:15]: variable names can be more generalized as the color can be changed. eg: error-color, success-color etc.

**Review by anasm11 — COMMENTED** [review]
I have suggested a change. Looks good to merge.

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
+.body {
+    font-family: var(--primary-font);
+}
+
+.para-span {
+
```
Comment (parvez-rashid) [inline components/alert/alert.css:6]: Remove the extra space i.e. empty new lines.

**Review by parvez-rashid — CHANGES_REQUESTED** [review]
Remove the extra spaces, rest looks good to merge.

File: components/alert/alert.css (hunk 2 above)
Comment (rdthetha) [reply components/alert/alert.css:6]: ok removed

File: component.css (hunk 1 above)
Comment (rdthetha) [reply component.css:15]: sure would change it!

**Pull request merged**